client/src/app/site/users/components/user-list/user-list.component.ts

```diff
@@ -69,4 +93,17 @@ export class UserListComponent extends ListViewBaseComponent<ViewUser> implement
     public onPlusButton(): void {
         this.router.navigate(['./new'], { relativeTo: this.route });
     }
+
+    // TODO save all data from the dataSource
+    public saveUserList(): void {
```

Comment:
Please use 'csv' in the function name, e.g. `csvExportUserList()`, because later we'll export the user list also in PDF. See function names in 2.3.x (`pdfExportUserList()`, `pdfExportUserAccessDataList()`, `csvExport())`: https://github.com/OpenSlides/OpenSlides/blob/stable/2.3.x/openslides/users/static/templates/users/user-list.html#L67

Also the menu button should show 'CSV', e.g. 'Export (CSV)'.